package com.siiet.Exception;

public class NestedDemo {
	public static void main(String[] args) {
		NestedTryCatch.check();
	}
}
